### ISSUE TITLE
Show banner with status

### ISSUE THREAD

**Issue by dianezhou96** [dianezhou96/lululedger#35]
Order deadline. Orders are autosaved; edit until order deadline. Make this banner closeable.

**Comment by dianezhou96** [comment]
Partially done. However, not connected to DB and isn't absolutely positioned. Ideally, also can show banner only once per session (if user closes it, shouldn't show up again).

**Comment by dianezhou96** [comment]
Could be related to #61

**Comment by dianezhou96** [comment]
Banner works fine. Make a new issue if want to do the improvements in previous comment.
